impl/client/src/main/javascript/web/pentaho/visual/config.js

```diff
@@ -834,7 +834,9 @@ define(function() {
         },
         apply: {
           extension: {
-            valuesVisible: true
+            valuesVisible: true,
```

Comment:
I suggest to use an approach to data labels similar to that of the Sunburst visualization, now or later.

See:
- https://github.com/pentaho/pentaho-platform-plugin-common-ui/blob/0b17995811f7c316c801315b06e445347516deec/impl/client/src/main/javascript/web/pentaho/visual/models/Sunburst.js#L67-L74
- https://github.com/pentaho/pentaho-platform-plugin-common-ui/blob/0b17995811f7c316c801315b06e445347516deec/impl/client/src/main/javascript/web/pentaho/ccc/visual/Sunburst.js#L70-L151

Reply:
And I've now circled back on the new `includeSizeInLabel` property, which offers the same functionality as the existing, above referenced, `labelsOption`. This new way of offering this functionality is not consistent with how it is offered in other vizs, especially, the sunburst viz.

Reply:
Thanks. I did required changes to be in consistent with Sunburst.